docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,165 @@
+         15. Grade 12 (no diploma)
+         16. Regular high school diploma
+         17. GED or alternative credential
+         18. Some college but less than 1 year
+         19. 1 or more years of college credit but no degree
+         20. Associate's degree
+         21. Bachelor's degree
+         22. Master's degree
+         23. Professional degree beyond a bachelor's degree
+         24. Doctorate degree
+
+   *  - MAR
+      - Marital status:
+         1. Married
+         2. Widowed
+         3. Divorced
+         4. Separated
+         5. Never married or under 15 years old
+
+   *  - OCCP
+      - Occupation. There are over 500 categories. Please see data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list of occupation codes.
+
+   *  - POBP
+      - Place of birth. There are over 200 categories, including the 50 US states and several countries. Please see the data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list.
+
+   *  - RELP
+      - Relationship:
+         0. Reference person
+         1. Husband or wife
+         2. Biological son or daughter
+         3. Adopted son or daughter
+         4. Stepson or stepdaughter
+         5. Brother or sister
+         6. Father or mother
+         7. Grandchild
+         8. Parent-in-law
+         9. Son-in-law or daughter-in-law
+         10. Other relative
+         11. Roomer or boarder
+         12. Housemate or roommate
+         13. Unmarried partner
+         14. Foster child
+         15. Other nonrelative
+         16. Institutionalized group quarters population
+         17. Noninstitutionalized group quarters population
+
+   *  - WKHP
+      - Usual hours worked per week in the past 12 months. Values are an integer from 1 to 99. Any hours above 99 are rounded down to 99
+
+   *  - SEX
+      - Sex code:
+         1. Male
+         2. Female
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PINCP. For generalizability, the integer value is provided.
+A threshold can be applied to PINCP to frame this as a binary classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PINCP
+      - Total income, denoted as an integer ranging from 104 to 1,423,000.
+
+
+An additonal column for the state code is also provided so users to easily filter the data to focus on subsets by state.
```

Comment:
```suggestion
An additional state code column allows users to easily filter the data to focus on subsets by state.
```

Reply:
Also 'easily' is splitting an infinitive (not the worst of sins, but I don't think it adds anything here), and is a bit redundant.

Reply:
Based on @romanlutz comment above, I'm changing the function so that the state code is no longer returned. So I'll remove this part from the documentation